extensions/script/python/PythonCreator.h

```diff
@@ -129,7 +129,8 @@ class PythonCreator : public minifi::core::CoreComponent {
       return "";
     }
     const auto python_package_path = std::filesystem::relative(pythonscript, basePath).parent_path();
-    std::vector<std::string> path_elements{python_package_path.begin(), python_package_path.end()};
+    std::vector<std::string> path_elements;
+    std::transform(python_package_path.begin(), python_package_path.end(), std::back_inserter(path_elements), [](const auto& path) { return path.string(); });
```

Comment:
```suggestion
    std::vector<std::string> path_elements;
    path_elements.reserve(std::distance(python_package_path.begin(), python_package_path.end()));
    std::transform(python_package_path.begin(), python_package_path.end(), std::back_inserter(path_elements), [](const auto& path) { return path.string(); });
```

Reply:
done in ce7b901964cb0c53365c40ba7d5f00fdd701665a